backend/quadcore/manager/auth.py

```diff
@@ -0,0 +1,38 @@
+from quadcore import oauth
+
+class AuthManager:
+
+    oauth_github = oauth.remote_app(
+        'github',
+        consumer_key='Iv1.2ce83a98d79af095',
+        consumer_secret='05316b10734d10fa53790d109f8ddc7484e8758e',
```

Comment:
api keys shouldn't be checked into repository - Note that this repository is public, so anyone can use this.